exodus_lambda/functions/origin_request.py

```diff
@@ -16,14 +17,66 @@ class OriginRequest(LambdaBase):
     def __init__(self, conf_file=CONF_FILE):
         super().__init__("origin-request", conf_file)
         self._db_client = None
+        self._sm_client = None
         self._cache = cachetools.TTLCache(
             maxsize=1,
             ttl=timedelta(
-                minutes=self.conf.get("config_cache_ttl", 2)
+                minutes=int(self.conf.get("config_cache_ttl", "2"))
             ).total_seconds(),
             timer=time.monotonic,
         )
 
+    @property
+    def db_client(self):
+        if not self._db_client:
+            self._db_client = boto3.client("dynamodb", region_name=self.region)
+
+        return self._db_client
+
+    @property
+    def sm_client(self):
+        if not self._sm_client:
+            self._sm_client = boto3.client(
+                "secretsmanager", region_name=self.region
+            )
+
+        return self._sm_client
+
+    @property
+    def secret(self):
+        if not self._cache.get("secret"):
+            arn = self.conf.get("secret_arn")
+            self.logger.warning("attempting to get secret %s", arn)
```

Comment:
These shouldn't be warnings, they might have been set that way during development, but debug makes more sense going forward.

Reply:
Actually, it seems we've chosen to use info for everything within lambda functions as we only look through these logs in CloudWatch when debugging.

Reply:
There's one debug statement and it's for an error. So, yeah, it seems logging scope is shifted for lambda.